.nvmrc

```diff
@@ -0,0 +1 @@
+v8.15.1
```

Comment:
Why not only 8 ? It should be enough ans follow patches upgrades

Reply:
C'est pour le dev du package. et notamment éviter les problème de version de npm. (d'où la version de 8 assez haute)

Reply:
Yes but you can only define `8` in your `.nvmrc` file, it will follow upgrades :)

Reply:
but it only consider the version you have installed